imod_coupler/drivers/dfm_metamod/mf6_utilities.py

```diff
@@ -0,0 +1,25 @@
+import numpy as np
+from numpy.typing import NDArray
+from xmipy import XmiWrapper
+
+
+class MF6Utilities:
+    @classmethod
+    def set_river_stages(
+        cls,
+        mf6_wrapper: XmiWrapper,
+        mf6_flowmodel_key: str,
+        mf6_river_pkg_key: str,
+        new_river_stages: NDArray[np.float_],
+    ) -> None:
+        """
+        sets the river stages in a modflow simulation to the provided values.
```

Comment:
That is useful for generating python docstrings: https://marketplace.visualstudio.com/items?itemName=njpwerner.autodocstring

We currently use the numpy conventions: https://numpydoc.readthedocs.io/en/v1.1.0/format.html

Reply:
Here I would have expected that the parameters are documented.

Reply:
done. Installed the plugin and set settings to numpy format.